truss/truss_config.py

```diff
@@ -29,8 +29,8 @@
 DEFAULT_SPEC_VERSION = "2.0"
 DEFAULT_PREDICT_CONCURRENCY = 1
 
-DEFAULT_CPU = "500m"
-DEFAULT_MEMORY = "512Mi"
+DEFAULT_CPU = "1"
```

Comment:
Ah gotcha. We wanted to make that change because the default values weren't super intuitive (500m? 512Mi?) but we weren't sure if Baseten required the buffer.

We should work on a way to make specifying resources more intuitive but it doesn't have to block this release.

Reply:
> Changing these default will map to something that's not the supported lowest instance type (which is 1cpu, 2Gi memory minus daemonsets, which will need some buffer.

I tried deploying this and it did deploy on the 1x2 instance